lua/lspconfig/dockerls.lua

```diff
@@ -8,7 +8,9 @@ configs[server_name] = {
   default_config = {
     cmd = { bin_name, '--stdio' },
     filetypes = { 'Dockerfile', 'dockerfile' },
-    root_dir = util.root_pattern 'Dockerfile',
+    root_dir = function(fname)
+      return util.root_pattern 'Dockerfile'(fname) or util.path.dirname(fname)
```

Comment:
I was thinking. Is there a point in setting this to anything but `util.path.dirname(fname)`?

Reply:
Not sure tbh.
I just went through their docs over at https://github.com/rcjsuen/dockerfile-language-server-nodejs, and I can't seem to find anything that seems dependent on this.